appmap/test/test_django.py

```diff
@@ -235,7 +230,11 @@ def server_start(env_vars_str):
     @staticmethod
     def server_stop():
         exec_cmd(
-            "ps -ef | grep -i 'manage.py runserver' | grep -v grep | awk '{ print $2 }' | xargs kill -9"
+            "ps -ef"
+            + "| grep -i 'manage.py runserver'"
+            + "| grep -v grep"
+            + "| awk '{ print $2 }'"
+            + "| xargs kill -9"
```

Comment:
Nice.  May as well do the same in `test_flask.py`.

Reply:
Done.